scripts/EN/Harry

```diff
@@ -1988,9 +1988,9 @@ LoadSecondEffect(0x0, 0x0, 0x0); // 0x23a8: 0000306e
 CleanScreen(); // 0x23b2: 00000064
 LoadBackground(0x16); // 0x23b6: 00001069
 DrawBackground(); // 0x23bc: 00000065
-ShowText((1107)"My one shot hits the wing, blasting a hole in it. The creature loses altitude as it falls to the floor."); // 0x23c0: 0000100c
+ShowText((1107)"My one shot hits the wing, W1blasting a large hole in it.¥nW1The creature then loses altitude and falls to the floor with a smack."); // 0x23c0: 0000100c
```

Comment:
Did a bit more research and found that kutyari-to is more onomatopoeia! I checked out some Japanese sites and they defined it as the sharp sound that happens when something breaks or is crumpled up. So I changed the last sentence to end with "...falls to the floor with a smack." to convey this.

Cell I97 also contains this change.